templates/pages/home.html

```diff
@@ -27,14 +28,17 @@
 <div class="main full">
     {{#if products.featured}}
         {{> components/products/featured products=products.featured columns=theme_settings.homepage_featured_products_column_count}}
+        {{{region name="page_content"}}}
```

Comment:
Could be better named to indicate its relation to Featured Products. Also, why is this below instead of above?